data/org.freedesktop.portal.Background.xml

```diff
@@ -110,19 +80,15 @@
       Sets the status of the application running in background.
 
       Supported keys in the @options vardict include:
-      <variablelist>
-        <varlistentry>
-          <term>message s</term>
-          <listitem><para>
-            A string that will be used as the status message of the application.
-            This should be a single line that can be presented to the user in a
-            list, not a full sentence or paragraph. Must be shorter than 96
-            characters.
-          </para></listitem>
-        </varlistentry>
-      </variablelist>
-
-      This method was added in version 2.
+
+      ``message`` (``s``)
+        A string that will be used as the status message of the application.
+        This should be a single line that can be presented to the user in a
+        list, not a full sentence or paragraph. Must be shorter than 96
+        characters.
+
+      .. important::
+         This method was added in version 2.
```

Comment:
IMO this should be moved up to the top, right now it breaks the rendering up in a confusing way.

Reply:
We should decide on a consistent style:

- what kind of admonition should be used: important? info?
- where should it be placed: before the text? After?

Reply:
I'd vote for info or note and placed before